.github/workflows/precommit.yml

```diff
@@ -16,7 +16,9 @@ jobs:
       fail-fast: false
 
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # tag=v4.2.2
```

Comment:
and why the specific hash tags? the tag should be fine, no?

Reply:
https://github.com/JustinGuese/python-openobserve/security/code-scanning/30